src/Envelope.php

```diff
@@ -5,7 +5,7 @@
 use lewiscowles\Rfc\NodeInterface;
 
 
-Final class Envelope implements NodeInterface {
+final class Envelope implements NodeInterface {
```

Comment:
specificity over caps... This really should come with a coding style issue and PR first

Reply:
It's the same explanation in the previous comment.